src/components/overview/feedback/Filelist.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Input } from "semantic-ui-react";
+import { getIPFSIp } from "../../application";
+
+export default class Filelist extends React.Component<any, any> {
```

Comment:
Change class name to `FileList` comply to naming conventions. Also rename the file name to `fileList.tsx` instead of `FileList.tsx`.

```suggestion
export default class FileList extends React.Component<any, any> {
```